projects/plugins/jetpack/modules/custom-post-types/comics.php

```diff
@@ -504,6 +504,16 @@ public function site_supports_comics() {
 			$supports_comics = true;
 		}
 
+		if ( $supports_comics ) {
+			$message = sprintf(
+				// translators: %s is a link to the WordPress.org documentation.
+				__( 'Jetpack no longer supports the Comics Post Type: %s', 'jetpack' ),
+				'https://jetpack.com/support/custom-content-types/#comics'
+			);
+
+			_deprecated_hook( 'Jetpack_Comic', 'jetpack-$$next-version$$', '', esc_html( $message ) );
```

Comment:
`_deprecated_hook` allows us to add an extra message.

Reply:
I like the approach with the message but placing it here generates a lot of lines in logs. What do you think about moving it to line 236 just after we attempt to register needed scripts?

Reply:
> What do you think about moving it to line 236 just after we attempt to register needed scripts?

I think that'd diminish the reach of the warning since it'd only be triggered upon visiting the frontend and not WPadmin.

> but placing it here generates a lot of lines in logs

It doesn't feel like a problem to me given how low profile this feature is (internally, it's only used in one retired theme) but I don't mind moving it if you do feel that's an issue.

Reply:
OK, that sounds reasonable to me.